java/ql/lib/semmle/code/java/JDK.qll

```diff
@@ -244,6 +280,8 @@ class MethodAccessSystemGetProperty extends MethodAccess {
   /**
    * Holds if this call has a compile-time constant first argument with the value `propertyName`.
    * For example: `System.getProperty("user.dir")`.
+   *
+   * Note: Better to use `semmle.code.java.environment.SystemProperty#getSystemProperty` instead.
    */
   predicate hasCompileTimeConstantGetPropertyName(string propertyName) {
```

Comment:
@smowton wondering if I should deprecate this in favor of the new way of finding all expressions that access system properties

Reply:
As of my changes, it's only ever now used in my code, it isn't used anywhere else in this repository.

Reply:
Also, at your request, I've improved my comment